<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSolveWindow</class>
 <widget class="QMainWindow" name="AddSolveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>701</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AddSolveWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>5</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Add Solve</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>2</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>571</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
